src/test/regress/sql/worker_range_partition.sql

```diff
@@ -21,22 +21,22 @@ ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1160000;
 \set Table_Part_02 lineitem_range_part_02
 \set Table_Part_03 lineitem_range_part_03
 
+\set File_Prefix   base/pgsql_job_cache/job_
+\set Table_File_00 :File_Prefix:JobId/task_:TaskId/p_00000
```

Comment:
Minor, this would be slightly more readable as:
```
\set File_Prefix   base/pgsql_job_cache
\set Table_File_00 :File_Prefix/job_:JobId/task_:TaskId/p_00000
```